dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -62,46 +63,53 @@
   private final ImmutableList<Path> inputClasspath;
   private final SyntheticRepository syntheticRepository;
   private final ClassLoader classLoader;
```

Comment:
Do we need both classloaders? If so, why?

Reply:
Good catch. `classLoader` is no longer needed and now removed.
(It used to play a role to construct argument list to lookup methods from Java class. The logic is no longer there)